### PR TITLE
Error handling

### LINKED ISSUE #97: Message when nothing displayed
When mimic can't connect to Aquilon and get information, it just displays a blank page on those tabs that need information from it. It would be nice if an error message was displayed explaining what's going on.

### PR DESCRIPTION
**Pull request by GooBall**
- Every view will now display a neat warning in the bottom right of the screen if that particular view can't connect or retrieve data from it's source.
- Fixes #97

### CODE REVIEW

File: inc/db-magdb-open.inc.php [hunk 1]
```diff
@@ -7,6 +7,6 @@
 
 #Connect to DB
 $PGLINK = pg_connect("host=$PG_HOST user=$PG_USER password=$PG_PASS dbname=$PG_NAME")
-or trigger_error("Unable to open connection to magDB", E_USER_ERROR);
+or error("Unable to conect to:", $PG_HOST);
```
Comment (jrha) [inline inc/db-magdb-open.inc.php]: con**n**ect

Do we really want to expose the hostname of the database server?
If we do, should we not also expose things such as the aquilon URL in later error messages?

File: inc/db-open.inc.php [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+
+$DB_BATCH_NAME    = $CONFIG['DB_GEN']['BATCH_NAME'];
+
+$DB_GEN_HANDLE = mysql_pconnect($DB_GEN_HOST, $DB_GEN_USER, $DB_GEN_PASS)
+or error("Unable to connect to:", $DB_GEN_HOST);
+
+$DB_BATCH_HANDLE  = $DB_GEN_HANDLE;
+
+mysql_select_db($DB_BATCH_NAME, $DB_BATCH_HANDLE)
+or error("Cannot use:", $DB_BATCH_NAME);
```
Comment (jrha) [inline inc/db-open.inc.php]: Does the error this generates make sense?

File: inc/error.inc.php [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<?php
+echo "<div class='error-zone'>$errors</div>"; // Displays errors in the bottom right of a view
```
Comment (jrha) [inline inc/error.inc.php]: Could we call this file something more meaningful like `error-zone` or `render-errors`?

Also, seeing as it is mostly html, how do you feel about this?

``` html
<div class='error-zone'><?php echo $errors ?></div>
```

Comment (GooBall) [inline inc/error.inc.php]: Yup, makes sense.

File: inc/db-open.inc.php (hunk 2 above)
Comment (GooBall) [inline inc/db-open.inc.php]: I'll make it a bit more descriptive.

File: inc/db-magdb-open.inc.php (hunk 1 above)
Comment (GooBall) [inline inc/db-magdb-open.inc.php]: Seeing as this isn't an error log dump, I'll make it show the name of the service instead of the full hostname.

**Pull request merged**